packages/shared/prisma/migrations/20240513082203_scores_unique_id_and_projectid_instead_of_id_and_traceid/migration.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[id,project_id]` on the table `scores` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropIndex
+DROP INDEX "scores_id_trace_id_key";
+
+-- CreateIndex
+CREATE UNIQUE INDEX "scores_id_project_id_key" ON "scores"("id", "project_id");
```

Comment:
this should be in its own migration file with `concurrently`

Reply:
done, moved to new migration